usaspending_api/search/delta_models/award_search.py

```diff
@@ -311,8 +319,36 @@
   COALESCE(transaction_fpds.place_of_performance_zip5, transaction_fabs.place_of_performance_zip5) AS pop_zip5,
   LPAD(CAST(CAST(REGEXP_EXTRACT(COALESCE(transaction_fpds.place_of_performance_congr, transaction_fabs.place_of_performance_congr), '^[A-Z]*(\\d+)(?:\\.\\d+)?$', 1) AS SHORT) AS STRING), 2, '0')
             AS pop_congressional_code,
-  -- TODO: include pop_congressional_code_current derivation
-  'TEST CUR POP CONGR AS' AS pop_congressional_code_current,
+  (CASE
+      WHEN (
+          transaction_fabs.place_of_performance_scope = 'Foreign'
+          OR transaction_fpds.place_of_perform_country_c <> 'USA'
+      ) THEN NULL
+      WHEN (
+          transaction_fabs.place_of_performance_scope = 'Multi-State'
+      ) THEN '90'
+      WHEN (
+          (transaction_fabs.place_of_performance_scope = 'State-wide'
+          OR transaction_fpds.place_of_perform_country_c = 'USA')
```

Comment:
Similar to my comment about for RL. Can these checks for `OR transaction_fpds.place_of_perform_country_c = 'USA'` (after the first one) be removed to simplify the query?

Reply:
Same as my comment above about resolving based on the `pop_state_cd` table, won't this always yield a result, essentially setting all records to a state-wide district?

Reply:
I can see the logic with Recipient Location CDs but in this case we're accounting for two different paths together (one being based on ppop scope for FABS, the other based on county code not being USA for FPDS or FABS). So I'd be worried if we took out the following USA checks for FPDS.

Say a record is FPDS, the ppop country code = 'USA', and it would derive for a state-wide code of '00'. Then when it hits the condition:
```
(
          (transaction_fabs.place_of_performance_scope = 'State-wide')
          AND pop_cd_state_grouped.congressional_district_no IS NOT NULL
      )
      ```
It would move on cause it doesn't meet the ppop scope (since that is null for FPDS) and set the condition to False.

Reply:
Okay, I think I see what you mean. Thanks for the clarification!